frontend/src/scenes/insights/EmptyStates/EmptyStates.tsx

```diff
@@ -51,7 +51,7 @@ export function InsightTimeoutState({
     return (
         <div className="insight-empty-state warning">
             <div className="empty-state-inner">
-                <div className="illustration-main height-auto">
+                <div className="illustration-main h-auto">
```

Comment:
I blame whoever reviewed it

![homer-hedge](https://user-images.githubusercontent.com/984817/225424618-10f82568-5e4d-4f46-b449-48b911ef6e77.gif)